src/Ocelot/Configuration/SecurityOptions.cs

```diff
@@ -1,17 +1,43 @@
 ﻿using System.Collections.Generic;
+using System.Linq;
+using NetTools;
```

Comment:
Let's reload and define namespace alias as the `IPAddressRange` name!
So, it is not clear what package is this namespace defined in!
```csharp
using IPAddressRange = NetTools;
```
Also we must add licensing info to this file as in docs of **IPAddressRange** GitHub repo!